### PR TITLE
update cli, install and supervisor docs to provide windows specific info

### PR DESCRIPTION
**Pull request by mwrock**
Signed-off-by: mwrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by mwrock** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/333541248).

File: www/source/partials/docs/_using-hab-packages.html.md.erb [hunk 1]
```diff
@@ -30,6 +30,13 @@ If your host machine is running Linux, do the following to run your packages:
     $ sudo hab start yourorigin/yourname
     ```
 
+You may use the same `hab start` command on Windows but omit the `sudo` command. However, you should be inside of an elevated shell. Also, not that the `hab` user is not necessary on Windows. If it is absent, services will run under the identity of the current user. If a `hab` user is present, you will need to provide its password via the`--password` argument:
```
Comment (cnunciato) [inline www/source/partials/docs/_using-hab-packages.html.md.erb]: Little typo: `Also, note that...`

File: www/source/partials/docs/_using-hab-packages.html.md.erb [hunk 2]
```diff
@@ -85,6 +94,15 @@ After you have built the hart package, do the following:
         $ sudo hab start your_origin/your_package
         ```
 
+On Windows this looks very similar with the the additional step of installing the Chocolatey package manager:
```
Comment (cnunciato) [inline www/source/partials/docs/_using-hab-packages.html.md.erb]: Looks like you've got an extra `the` tagging along there. 😄

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/333541248).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by mwrock** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/333546161).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/333546161).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by mwrock** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/333550645).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/333550645).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by mwrock** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/333553384).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/333553384).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/partials/docs/_commands-cli.html.md.erb [hunk 3]
```diff
@@ -585,6 +585,7 @@ Builds a Plan using a Studio
 
 **FLAGS** 
 
+    -w, --windows    Use a Windows Studio instead of a Docker Studio (only available on Windows)
```
Comment (fnichol) [inline www/source/partials/docs/_commands-cli.html.md.erb]: This is good, although given our current process (which is to copy/paste CLI output when refreshing these docs), I wouldn't be surprised if these get left out the next time all commands get refreshed. Clearly we do need a better process here, but just making a note for the time-being.

File: www/source/partials/docs/_using-hab-packages.html.md.erb [hunk 4]
```diff
@@ -145,7 +170,7 @@ Sometimes you need to stop a running service for a period of time, for example d
 $ hab svc stop core/redis
 ```
 
-## Starting a Loaded Stopped Service
+## Starting a Loaded Stopped Servicestrat
```
Comment (fnichol) [inline www/source/partials/docs/_using-hab-packages.html.md.erb]: Maybe typo here?

**Comment by fnichol** [comment]
Hilarious that our bot wasn't going to merge this

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/334830734).

**Comment by fnichol** [comment]
I think it's going to make it this time!

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/334830734).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**